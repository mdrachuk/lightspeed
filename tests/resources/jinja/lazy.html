<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Lazy</title>
</head>
<body>
{{ lazy }}
</body>
</html>