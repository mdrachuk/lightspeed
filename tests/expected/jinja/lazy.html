<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Lazy</title>
</head>
<body>
Hello there! lazy.html
</body>
</html>